src/dbnode/integration/integration.go

```diff
@@ -295,6 +318,14 @@ func newDefaultBootstrappableTestSetups(
 	}
 }
 
+type dynamicTopoMapProviderBullshit struct {
+	setup *testSetup
+}
+
+func (d *dynamicTopoMapProviderBullshit) TopologyMap() (topology.Map, error) {
```

Comment:
lol this can just be a method on `*testSetup` no?

Reply:
...yes that would better. That was written in anger, will make it a method